integrations/facebook-pixel/test/index.test.js

```diff
@@ -1344,6 +1343,38 @@ describe('Facebook Pixel', function() {
         );
         assertEventId(window.fbq);
       });
+
+      it('should send facebook pixel properties for product viewed', function() {
+        analytics.track('Product Viewed', {
+          currency: 'USD',
+          products: [
+            {
+              product_id: '507f1f77bcf86cd799439011',
+              name: 'Monopoly: 3rd Edition',
+              price: 19,
+              quantity: 1
+            }
+          ]
+        });
+        analytics.called(
+          window.fbq,
+          'trackSingle',
+          options.pixelId,
+          'ViewContent',
+          {
+            content_ids: ['507f1f77bcf86cd799439011'],
+            content_type: ['product'],
+            content_name: '',
```

Comment:
This test wouldn't pass after we implement the updated logic correctly. After updating the logic, we would map `name` in the track event to `content_name` in the Facebook Standard event. So, the value of `content_name` here should be "'Monopoly: 3rd Edition'".